apps/api_web/lib/api_web/views/prediction_view.ex

```diff
@@ -204,6 +206,12 @@ defmodule ApiWeb.PredictionView do
     |> String.upcase()
   end
 
+  def revenue(%{revenue: atom}) do
+    atom
+    |> Atom.to_string()
+    |> String.upcase()
```

Comment:
question: isn't the atom already uppercase? In which case you can pass it directly (I think).